### ISSUE TITLE
Error at line 114 #include, fileinstalls.ahk and 'unsupported resolution' at a supported resolution (can't find file)

### ISSUE THREAD

**Issue by LeafFresh** [Nigh/Genshin-fishing#50]
When running the script it says 'Error at line 114 #include, fileinstalls.ahk' which can easily be fixed by commenting it or removing the line entirely (it's not used anywhere anyway)

After that when I run the script it will tell me I have a unsupported resolution 1920 x 1080 and other reslolutions I tried but it is in the supported resoltion list, so likely something else, as I went debugging I found out that the A_Temp variable checks on your appdata/local/temp location on your pc, and it wants to check if images in the folder genshinfishing/winW winH exists. except I didn't have my folder in the TEMP folder (or it wasn't generated) and my folder was called Genshin-fishing-main and not genshinfishing. 
And the resolution folders don't exist in the main folder but in the assets folder.  I could fix these problems myself and now can use the script as intended.

How I solved it in 2 ways:  
1. Putting my files in the TEMP folder and renaming everything as the code wanted and adding /assests/ after /genshinfishing/ where neccesarry. but since this is the temp folder I it can get easily removed by your pc (as it did for me after closing the script).

2. removing A_Temp in the code and chaging "/genshinfishing/" to "assets/" which also solved the issue and now the script was wroking properly!

Maybe didn't install it properly? or using an incompatible version of AutoHotKey? 
Maybe the code was supposed to generate files in the Temp folder and it's not working properly on my device?

I hope this is mainly an issue at my end so not other people are experienceing this.

Have a wonderfull day!

**Comment by Nigh** [comment]
This script is designed to run after being compiled into a binary. You should run dist.ahk, which will automatically package a fileinstalls file and compile it into a binary. Running the compiled binary is necessary to correctly release the files into the temporary directory.

**Comment by LeafFresh** [comment]
I tried this and it worked! it created what it's supposed to! 
I was really confused since it didn't say any of this in the readme.md file (about dist.ahk)
But now I realized that I am simply running the script from the sourcecode and not the actual download (with just the exe)
I didn't think much of it since the video I found the script on did the exact same thing (downloaded source code) except didn't run 'disk.ahk' and directly ran 'GenshingFishing.ahk'! it's an older video, about 2 years old so out-dated ofcourse....  
We all make mistakes so I can only learn from this! Besides it was fun for me learning how the script worked and debugging it, first time doing anything with ahk related! 

Non-theless I really like this script and I appriciate your work on this! and thank you for helping me!

Have a good day!
